enclave-manager/web/src/client/enclaveManager/KurtosisClientContext.tsx

```diff
@@ -70,7 +70,7 @@ export const KurtosisClientProvider = ({ children }: PropsWithChildren) => {
             throw Error(`Port ${port} is not a number.`);
           }
 
-          const jwtToken = Cookies.get("kurtosis");
+          const jwtToken = Cookies.get("_kurtosis_jwt_token");
```

Comment:
@Dartoxian this will need a new build. Approving so you can merge when new build